src/v2/view-builder/views/TerminalView.js

```diff
@@ -5,7 +5,7 @@ import BaseFooter from '../internals/BaseFooter';
 const Body = BaseForm.extend({
   title () {
     const msg = this.options.appState.get('terminal').message || {};
-    return msg.value || 'You can close this window';
+    return msg.message || 'You can close this window';
```

Comment:
This is for an API that is being made with this PR https://github.com/okta/okta-core/pull/36839

Reply:
can we update API mocks as well?

Reply:
yeah, just waiting to verify with the API changes.